crates/ty_completion_eval/truth/auto-import-skips-current-module/main.py

```diff
@@ -0,0 +1,3 @@
+Kadabra = 1
+
+Kada<CURSOR: Kadabra>
```

Comment:
To make sure I understand the test case. This used to suggest `Kadabra` twice, once as local symbol and once as import?